examples/minimal-react/src/components/App.tsx

```diff
@@ -1,95 +1,64 @@
+export const { useApi, useTracks, useStatus, useConnect, JellyfishContextProvider } = create<
+  PeerMetadata,
+  TrackMetadata
+>();
 
 export const App = () => {
-  // Create the connect function
-  const connect = useConnect();
-
-  // Get the full state
-  const remoteTracks = useSelector((snapshot) => Object.values(snapshot?.remote || {}));
-
-  // Get the webrtcApi reference
-  const webrtcApi = useSelector((snapshot) => snapshot.connectivity.api);
-
-  // Get jellyfish client reference
-  const jellyfishClient = useSelector((snapshot) => snapshot.connectivity.client);
-
-  useEffect(() => {
-    const disconnect = connect({
-      peerMetadata: { name: "peer" },
-      token: peerToken,
-    });
-
-    return () => {
-      // Disconnect the peer when the component unmounts
-      disconnect();
-    };
-  }, [connect]);
-
-  useEffect(() => {
-    async function startScreenSharing() {
-      // Check if webrtc is initialized
-      if (!webrtcApi) return console.error("webrtc is not initialized");
+  const [token, setToken] = useState("");
 
-      // Create a new MediaStream to add tracks to
-      const localStream: MediaStream = new MediaStream();
-
-      // Get screen sharing MediaStream
-      const screenStream = await navigator.mediaDevices.getDisplayMedia(SCREEN_SHARING_MEDIA_CONSTRAINTS);
-
-      // Add tracks from screen sharing MediaStream to local MediaStream
-      screenStream.getTracks().forEach((track) => localStream.addTrack(track));
-
-      // Add local MediaStream to webrtc
-      localStream.getTracks().forEach((track) => webrtcApi.addTrack(track, localStream, { type: "screen" }));
-    }
-
-    const onJoinSuccess = (peerId: string, peersInRoom: Peer[]) => {
-      console.log("join success");
-      console.log("peerId", peerId);
-      console.log("peersInRoom", peersInRoom);
-
-      // To start broadcasting your media you will need source of MediaStream like camera, microphone or screen
-      // In this example we will use screen sharing
-      startScreenSharing();
-    };
-
-    // You can listen to events emitted by the client
-    jellyfishClient?.on("joined", onJoinSuccess);
-
-    return () => {
-      // Remove the event listener when the component unmounts
-      jellyfishClient?.off("joined", onJoinSuccess);
-    };
-  }, [jellyfishClient, webrtcApi]);
+  const connect = useConnect();
+  const api = useApi();
+  const status = useStatus();
+  const tracks = useTracks();
 
-  // Render the remote tracks from other peers
   return (
-    <>
-      {remoteTracks.map(({ tracks }) => {
-        return Object.values(tracks || {}).map(({ stream, trackId }) => (
-          <VideoPlayer key={trackId} stream={stream} /> // Simple component to render a video element
-        ));
-      })}
-    </>
+    <div style={{ display: "flex", flexDirection: "column" }}>
+      <input value={token} onChange={(e) => setToken(() => e?.target?.value)} placeholder="token" />
+      <button
+        onClick={() => {
```

Comment:
I think that an option to disconnect would be quite useful here. This is a matter of simple check on server status and swapping the function into useDisconnect() function. It's always nice to show it is viable in API